frontend/testing/playwright/tests/data-model/data-model.spec.ts

```diff
@@ -55,46 +56,40 @@ test('Allows to add a data model, include an object with custom name and fields
     await dataModelPage.tabOutOfNameField();
     const newNameFieldValue = await dataModelPage.getNameFieldValue();
     expect(newNameFieldValue).toEqual(newName);
-    await dataModelPage.checkThatTreeItemProperyExistsOnScreen(newName);
+    await dataModelPage.checkThatTreeItemPropertyExistsOnScreen(newName);
   };
 
   // Rename the object
   const treeItemTestName: string = 'test';
   await replaceName0WithNewTextValue(treeItemTestName);
   await dataModelPage.clickOnTreeItemProperty(treeItemTestName);
 
-  // Helper function to add a new field to the test object added
-  const addFieldToTheTestNode = async () => {
+  // Helper function to add a new property to the test object added
+  const addPropertyToTheObjectNode = async (propertyName: TextKey) => {
```

Comment:
Based on my comment above, we can adjust like this:
```suggestion
  const addPropertyToTheObjectNode = async (propertyName:  'string' | 'number') => {
```